### ISSUE TITLE
[A11y]Keyboard Focus is not visible in some locations

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9416]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1743910

Environment Details:
App name: NuGet Service
#URL: [https://www.nuget.org](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=05%7c01%7cv-nitipi%40microsoft.com%7c9a84307e7aa744a4b24208dafa5ca2fe%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638097573052590837%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=C8Eab3i6ixAmqsOjG4AxNT36z9WZQiXPbjDDkMcS5ME%3D&reserved=0)

OS and Browser Details:
Edition Windows 11 Enterprise Insider Preview
OS build 25275.1000.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)

Repro Steps:
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Select “API Keys” from the Profile Dropdown and hit enter.
Navigate to "Create" button and hit enter.
Enter Values in "Key Name" and "Glob Pattern" field and hit enter on Create button.
Navigate to "Manage" control and hit enter.
Observe whether "edit, Regenerate, delete" keyboard focus is clearly visible or not visible.

Similar issue is observed for below screen also.

Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and hit enter.
Navigate to "Owners" section present on bottom right of the screen.
Observe whether "Profile" keyboard focus is clearly visible or not visible.

Actual Result:
Keyboard focus on "Edit, Regenerate, delete" is not clearly visible.

Expected Result:
Keyboard focus should be completely visible on the control wherever focus lands. It should be visible on all four sides on "Edit, Regenerate, Delete".

Notes:
The display (inline) and the packing of the elements around these appear to be covering the area that they keyboard focus border is highlighting. Unfortunately, it is not as easy as adding additional padding/margin around the elements as the inline display prevents some of the height changes. We will need to experiment with this a bit more to figure out a view with enough spacing to be able to see our keyboard focus.

**Comment by lyndaidaii** [comment]
duplicated: ttps://github.com/NuGet/Engineering/issues/4827
